src/gdk.jl

```diff
@@ -68,6 +68,86 @@ baremodule GdkKeySyms
   const Hyper_R = 0xffee
 end
 
+baremodule GdkAtoms
```

Comment:
where are you getting all these from? Gtk itself appears to only defines "GDK_NONE"

PyGTK defines a few extras (https://developer.gnome.org/pygtk/stable/class-gdkatom.html), but only because it has computed the value at runtime (https://developer.gnome.org/gdk3/stable/gdk3-Properties-and-Atoms.html#gdk-atom-intern)